week-1-ts-ecosystem/w01e03-object-oriented-structure/package.json

```diff
@@ -1,5 +1,24 @@
 {
   "name": "w01e03-object-oriented-structure",
+  "author": "Dariusz Gala",
   "version": "1.0.0",
-  "scripts": {}
+  "scripts": {
+    "watch": "nodemon src/main.ts",
+    "build": "tsc --project tsconfig.build.json"
+  },
+  "devDependencies": {
+    "@types/uuid": "^8.3.4",
+    "@typescript-eslint/eslint-plugin": "^5.38.0",
+    "@typescript-eslint/parser": "^5.38.0",
+    "amd-loader": "^0.0.8",
+    "eslint": "^8.24.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-import": "^2.26.0",
+    "nodemon": "^2.0.20",
+    "prettier": "^2.7.1",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.8.3",
+    "uuid": "^9.0.0"
+  },
+  "type": "module"
```

Comment:
`npm link` powinien być puszczony z opcja `--save` tak żebyśmy tutaj widzieli zależność, zobacz:

```
npm link w01e03-object-oriented-structure --save
```

wtedy w `package.json` jest wyraźnie:
```json
  "dependencies": {
    "w01e03-object-oriented-structure": "file:../w01e03-object-oriented-structure"
  }
```

A to bardzo ważne żeby cały config - był ok dla projektu dla innych developerów. (lub dla Ciebie z przyszłości :))